src/widget/wtrackmenu.cpp

```diff
@@ -1802,10 +1841,16 @@ void WTrackMenu::slotRemoveFromDisk() {
     }
     // Else show a message with a list of tracks that could not be deleted.
     QLabel* notDeletedLabel = new QLabel;
-    notDeletedLabel->setText(
-            tr("The following %1 files could not be deleted from disk")
-                    .arg(QString::number(
-                            tracksToKeep.length())));
+    QString msgText;
+    if (m_pTrackModel) {
+        msgText =
+                tr("The following %1 files could not be deleted from disk")
+                        .arg(QString::number(
+                                tracksToKeep.length()));
+    } else {
+        msgText = tr("Track file could not be deleted from disk.");
```

Comment:
Do we also shoe the not deleted track file? 
If yes, the text needs to reflect that.

Reply:
Yes, remaining file(s) are listed in the dialog.
I reworded the single file message to match the multiple files message.